modules/migration-changing-migration-plan-limits.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-changing-migration-plan-limits_{context}']
+= Changing migration plan limits for large migrations
+
+A single migration plan has the following limits for guidance when performing large scale migrations:
+
+* 10 namespaces. If this limit is exceeded, an error message appears and you cannot create a migration plan.
+* 100 Pods. If this limit is exceeded, a warning message appears.
+* 100 persistent volumes. If this limit is exceeded, a warning message appears.
+
+You can change the default limits by updating the following values in the Migration controller CR:
+
+[source,yaml]
```

Comment:
@jwmatthews 

Is this sufficient or do we need to include a procedure for editing the CR yaml? I don't have time to set up a 4.2 cluster today.

Reply:
I think below is sufficient just showing the values to change.

If you wanted to show how to change this, an example would be:

$ oc get migrationcontroller -n openshift-migration
NAME                   AGE
migration-controller   5d19h

$ oc edit migrationcontroller -n openshift-migration